### DISCUSSION TITLE
Add a way to open Quick Terminal from the command line

### DISCUSSION THREAD

**Discussion by susl** [ghostty-org/ghostty#3795] [Feature Requests, Ideas] [answered]
I'm using skhd to manage most of my system-wide shortcuts and I would like to be able to add a Quick Termanal there, rather than adding Accessibility permission to the new app (Ghostyy) and managing a keybinding there. 

I tried searching for a command (similar to other "+..") commands for ghostty to trigger quick terminal, but it seems that it isn't there.

**Comment by JakeWharton** [comment, marked as answer]
Should be part of #2353

**Comment by susl** [comment]
Hey! Yeah, seems reasonable to me. Please close this one. Do I need to add
any details to #2353?

**Reply by JakeWharton** [reply]
Probably not. I would expect the quick terminal to get treatment you desire. If you want to add a comment it certainly wouldn't hurt.

**Comment by rbo** [comment]
For me the following config works pretty well with skhd & yabai:
```bash
%  grep Ghostty ~/.config/skhd/skhdrc
alt - return : open -n /Applications/Ghostty.app

% cat ~/.config/ghostty/config
macos-titlebar-style = hidden
theme = "Apple System Colors"
font-family = "SauceCodePro NFM Light"
font-size=18
cursor-style = bar
background-opacity = 0.9
background-blur-radius = 20

keybind = global:ctrl+grave_accent=toggle_quick_terminal
```

**Reply by jamkich** [reply]
its decent solution, but that is opening new instance of app, not window like in other terminals.

**Reply by rbo** [reply]
Agree, I recognize after the post as well. I added 
```
quit-after-last-window-closed = "true"
```
to my ghosty config to close the application after the terminal is closed.

**Reply by rbo** [reply]
Having multi instances is not bad at all, I can navigate easy with command+tab between Terminal windows.

**Reply by jamkich** [reply]
Actually it's true, but for me it's just not looking good lol

**Reply by yoshimi-I** [reply]
I am using Ghostty and would like to display the quick terminal overlay on top of an active application (e.g., a browser like Google Chrome) with transparency.

Is there a way to achieve this?

**Reply by jamkich** [reply]
@yoshimi-I do you mean titlebar in ghostty window? if yes, for macos its *macos-titlebar-style*. If not, please let me know

**Comment by susl** [comment]
Yeah, but it does require special permissions from MacOS, to use the global
prefix

**Reply by rbo** [reply]
Yes right, Settings -> Privacy & Security -> Accessibility 
![Screenshot 2024-12-30 at 21 26 30](https://github.com/user-attachments/assets/2153ee0b-8ee6-479c-926a-6f2a9ae53413)
